CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.65.0
```

Comment:
You don't need to specify a version here, just write `## newVersion` and I will bump a version when I decided to release.
Please read the [contributing guideline](https://github.com/imaNNeo/fl_chart#contributing) carefully

Reply:
Merged with master and it is now under `##nextVersion`